Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -195,6 +195,14 @@ PropertyID property_id_from_string(StringView string)
     });
 
     generator.append(R"~~~(
+    if (string.starts_with("--"sv) && string != "--" && !is_ascii_digit(string.to_string()[2])) {
+        auto forbidden_characters = "\0\t\r\n !\"#$%&\'()*+,./:;<=>?[]\\^´`\u0040"sv;
+        for (auto character : forbidden_characters) {
```

Comment:
This looks like it will be quite slow, scanning the entire property value  20+ times. Is there an `is_any_of` helper on StringView or as a free function that we can use here? I feel like I've seen that pattern before in LibWeb. Though when the property name is smaller than the forbidden characters it ends up being a wash 🤔

Reply:
There is such a helper but I missed it.